src/backend/_scss/application.scss

```diff
@@ -481,7 +481,7 @@ a {
 .arrow {
   height: 10px;
   left: 6px;
-  margin-top: -2.3%;
+  margin-top: -9px;
```

Comment:
@aayusharora I am sorry. While working with other issues, the changes made in the gh-pages due to this pull request got lost and what you saw was the previous version without any alteration. I have updated my gh-pages to contain the changes done in this pull request.
The X-axis scroll bar is almost gone. It's still there but it's range is almost negligible.  Meaning that you can easily view the whole content without having to scroll to the right. To completely remove it we have to add the following properties to the code
```
body {
 overflow: -moz-scrollbars-vertical;
 overflow-x: hidden;
 overflow-y: scroll;
} 
```
I wasn't too sure about adding it so left it up for your review. About the margin values for the arrow, I tried different values both relative in percentages and absolute in pixels and the -10px seemed to work well on all the screens. We can change it if there is some issue with it. Thanks.
Please review again.
Link to updated gh-pages
https://princu7.github.io/websiteTest/speakers.html